Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go

```diff
@@ -251,6 +251,14 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		}
 	}
 
+	if !s.scc.AllowEmptyDirVolumePlugin {
+		for _, v := range pod.Spec.Volumes {
+			if v.EmptyDir != nil {
+				allErrs = append(allErrs, fielderrors.NewFieldInvalid("VolumeMounts", v.Name, "EmptyDir Volumes are not allowed to be used"))
```

Comment:
Error creating: Pod "testrc-" is forbidden: unable to validate against any security context constraint: [provider restricted: .spec.containers[0].securityContext.VolumeMounts: invalid value 'container-data', Details: EmptyDir Volumes are not allowed to be used]

I changed it to this now:

Error creating: Pod "testrc-" is forbidden: unable to validate against any security context constraint: [provider restricted: .spec.containers[0].securityContext.volumes[0]: invalid value 'emptyDir', Details: EmptyDir volumes are not allowed to be used]

I also adjusted the HostPath error just above where I got the bad example originally.